app/uk/gov/hmrc/vatapi/resources/VatReturnsResource.scala

```diff
@@ -68,7 +66,8 @@ object VatReturnsResource extends BaseController {
   }
 
   def retrieveVatReturns(vrn: Vrn, periodKey: String): Action[AnyContent] =
-    Action.async { implicit request =>
+    APIAction(vrn).async { implicit request =>
+      logger.debug(s"[VatReturnsResource] [retrieveVatReturns] Retrieve VAT returns for the VRN : $vrn")
```

Comment:
This is picking up the logger from BaseResource.  We can make this the standard logger or keep the resource specific logger they had created previously.